src/Init/Meta.lean

```diff
@@ -115,6 +115,8 @@ def getRoot : Name → Name
 
 @[export lean_is_inaccessible_user_name]
 def isInaccessibleUserName : Name → Bool
+  -- note: this uses `✝` (`\u{271D}`) whereas everywhere else in the code
+  -- we use the text-variant selector `✝︎` (`\u{271D}\u{FE0E}`).
```

Comment:
Maybe also say _why_ we use something else here?

```suggestion
  -- we use the text-variant selector `✝︎` (`\u{271D}\u{FE0E}`).
  -- This is easier simply because we don't have `s.hasSubstring` in core.
```